src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -342,10 +342,10 @@ class MySimulation : public SimulationHelper<simDim>
         if( step != 0 )
         {
             namespace nvfct = PMacc::nvidia::functors;
-            (*pushBGField)( fieldE, nvfct::Sub(), fieldBackgroundE(fieldE->getUnit()),
-                            step, fieldBackgroundE::InfluenceParticlePusher);
-            (*pushBGField)( fieldB, nvfct::Sub(), fieldBackgroundB(fieldB->getUnit()),
-                            step, fieldBackgroundB::InfluenceParticlePusher);
+            (*pushBGField)( fieldE, nvfct::Sub(), FieldBackgroundE(fieldE->getUnit()),
```

Comment:
that change is correct, but since it is highly risky for merge conflicts and is unrelated to the functionality of the pull please do not change it at the same time as you add twts.